lib/queue_config.rb

```diff
@@ -0,0 +1,41 @@
+require_relative './random_tools.rb'
+
+module Upaya
+  module QueueConfig
+    # rubocop:disable Metrics/MethodLength
+
+    # Known acceptable values for config.active_job.queue_adapter
+    KNOWN_QUEUE_ADAPTERS = %i[sidekiq inline async].freeze
+
+    # Select a queue adapter for use, including possible random weights as
+    # defined by Figaro.env.queue_adapter_weights (a JSON mapping from queue
+    # adapters to integer weights)..
+    def self.choose_queue_adapter
+      adapter_config = Figaro.env.queue_adapter_weights
```

Comment:
Did you mean to add an entry in `application.yml.example` for `queue_adapter_weights`?

Reply:
I wanted to leave it as sidekiq by default, so that's not necessary, but it could be useful to have an example.

Reply:
By including it in `application.yml.example`, it tells the release manager that there are new configs to add to our environments. It also shows clearly what the expected format is for the entry. Without an example, it wouldn't be obvious what needs to be added to the config once we decide we want to add other adapters.

Reply:
Added